README.md

```diff
@@ -24,17 +30,17 @@ config. Here is an example of it in use:
 ```yml
 name: Create envfile
 
-on: [push]
+on: [ push ]
 
 jobs:
 
   create-envfile:
  
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
  
     steps:
     - name: Make envfile
-      uses: SpicyPizza/create-envfile@v1
+      uses: SpicyPizza/create-envfile@master
```

Comment:
I think I would be more interested in keeping this pinned to a version, and changing this line on release. I don't want a release with regressions to break people's CI workflows. Can you set it to 1.2 for now?

Reply:
well, then in the file README.md it is already worth removing versions

Reply:
Ya, I wouldn't mind cleaning this up all the way.